tests/fr/lock_HassTurnOff.yaml

```
@@ -1,31 +1,88 @@
 language: fr
 tests:
+  # name
   - sentences:
-      - "déverrouille la porte d'entrée"
+      - déverrouille la porte d'entrée
+      - déverrouiller la porte d'entrée
```

```suggestion
      - déverrouiller porte d'entrée
```
So we test another case